src/components/academies/GoToAcademy.jsx

```diff
@@ -0,0 +1,63 @@
+import { Link } from 'react-router-dom';
+import { Alert, Button, Skeleton } from '@openedx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { Error } from '@openedx/paragon/icons';
+import React from 'react';
+import useEnterpriseCustomer from '../app/data/hooks/useEnterpriseCustomer';
+import { useAcademies } from '../app/data';
+
+const GoToAcademy = () => {
+  const { data: academies, isError: fetchError, isLoading } = useAcademies();
+  const { data: enterpriseCustomer } = useEnterpriseCustomer();
+
+  if (isLoading) {
+    return (
+      <>
+        <div className="sr-only">Loading...</div>
+        <Skeleton height={100} />
+      </>
+    );
+  }
+
+  if (fetchError || academies?.length !== 1) {
```

Comment:
[clarification] Is it necessary to render an error state here when `academies` does not have a length of 1? I wonder if a more user-friendly fallback would be to fallback to rendering the default empty state messaging (similar to the other comment around falling back to keeping "Find a Course" in the header in the event of zero or more than one academy existing for a customer with `enableOneAcademy` enabled).

Rationale: If a customer has `enableOneAcademy` enabled, but ends up with 2+ (or zero) academies configured without someone updating the `enableOneAcademy` flag, learners would start seeing error messages when a more graceful fallback might be the default empty state messaging. Doing that would also result in a slightly cleaner implementation because the check on `academies.length === 1` could be added directly within `CourseEnrollmentsEmptyState` instead of within `GoToAcademy`, e.g.:

```jsx
if (enterpriseCustomer.enableOneAcademy && academies.length === 1) { ... }
```